src/common/messages.cpp

```diff
@@ -53,6 +53,39 @@ const std::vector<std::pair<std::string, FeeEstimateMode>>& FeeModeMap()
     return FEE_MODES;
 }
 
+std::string FeeModeInfo(const std::pair<std::string, FeeEstimateMode>& mode)
+{
+    const char* info;
+    switch (mode.second) {
+        case FeeEstimateMode::UNSET:
+            info = "no mode set\n";
+            break;
+        case FeeEstimateMode::ECONOMICAL:
+            info = "Economical estimates use a shorter time horizon, making them more\n"
+                   "responsive to short-term drops in the prevailing fee market. This mode\n"
+                   "potentially returns a lower fee rate estimate.\n";
+            break;
+        case FeeEstimateMode::CONSERVATIVE:
+            info = "Conservative estimates use a longer time horizon, making them\n"
+                   "less responsive to short-term drops in the prevailing fee market. This mode\n"
+                   "potentially returns a higher fee rate estimate.\n";
+            break;
+        default:
+            // no default case, so the compiler can warn about missing cases
+            assert(false);
+    }
+    return strprintf("\"%s\" \n%s", mode.first, info);
+}
+
+std::string FeeModesDetail()
```

Comment:
I think this new format is a little difficult to read with the modes interleaved with their descriptions:

```
2. estimate_mode    (string, optional, default="economical") The fee estimate mode.
                    "unset"
                    no mode set
                    "economical"
                    Economical estimates use a shorter time horizon, making them more
                    responsive to short-term drops in the prevailing fee market. This mode
                    potentially returns a lower fee rate estimate.
                    "conservative"
                    Conservative estimates use a longer time horizon, making them
                    less responsive to short-term drops in the prevailing fee market. This mode
                    potentially returns a higher fee rate estimate.
```

IMO it may be clearer to switch to something more like this:

```
2. estimate_mode    (string, optional, default="economical") The fee estimate mode.
                    Available modes: "economical", "conservative", "unset"

                    Economical estimates use a shorter time horizon, making them more
                    responsive to short-term drops in the prevailing fee market. This mode
                    potentially returns a lower fee rate estimate.

                    Conservative estimates use a longer time horizon, making them
                    less responsive to short-term drops in the prevailing fee market. This mode
                    potentially returns a higher fee rate estimate.
```

Reply:
Taken